lib/topological_inventory/orchestrator/deployment_config.rb

```diff
@@ -72,6 +72,24 @@ def delete_in_openshift
         logger.info("[OK] Deleted DeploymentConfig #{self}")
       end
 
+      def recreate_in_openshift
+        delete_in_openshift
+        retry_count = 0
+
+        until retry_count == 3
```

Comment:
Probably overkill, but just asking. Is there any reason this should either be an env var? ie will 3 be too many?  Not enough?

Or at the very least should potentially be a constant for better exposure?

Reply:
Yeah - I like the idea of changing it to a constant. 30 seconds should be enough time for the resource to hopefully disappear, though I'm not quite sure we'd need it as an ENV var. I'm not sure we'd ever have to increase it.